cli/providers/aws/cf/create.go

```diff
@@ -17,6 +17,10 @@ func Create(name, template string, config *aws.Config) (string, error) {
 				Key:   aws.String("i2kit"),
 				Value: aws.String("alpha"),
 			},
+			&cloudformation.Tag{
```

Comment:
The tags have these restrictions:

- Maximum key length—127 Unicode characters in UTF-8
- Maximum value length—255 Unicode characters in UTF-8

do you think that we should enforce those here? Maybe chop the string if it's longer than that?

Reply:
Yes, we have to validate it. We also need to validate to have valid cnames for service discovery.
I would focus on validation on a different PR when defining the dns behavior.

Reply:
@rberrelleza 
https://trello.com/c/2Kd0k4za/61-validate-service-names-taking-into-account-cf-validator-and-dns-search-limitations